website/netlify.toml

```diff
@@ -74,6 +74,12 @@
   status = 301
   force = true
 
+[[redirects]]
+  from = "/docs/v0.7/*"
+  to = "https://v0-7--vector-project.netlify.com/docs/:splat"
```

Comment:
I like this strategy. :) Let's you avoid having to carry around docs for old versions in `master` forever.